docs/release-notes/8.0.34-26.md

```diff
@@ -0,0 +1,150 @@
+# Percona Server for MySQL 8.0.34-26 (2023-09-0)
+
+Get started with [Quickstart Guide for Percona Server for MySQL](../quickstart-overview.md).
+
+[Percona Server for MySQL 8.0.34-26](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.34 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-34.html) in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a freely available, fully compatible, enhanced, and open source drop-in replacement for any MySQL database. It provides superior and optimized performance, greater scalability, and availability, enhanced backups, increased visibility and instrumentation.
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide better performance and concurrency for their most demanding workloads.
+
+## Release highlights
+
+* Percona Server for MySQL 8.0.34-26 implements the [Audit Log Filter plugin](../audit-log-filter-overview.md) that allows you to monitor, log, and block a connection or query actively executed on the selected server. 
```

Comment:
@oleksandr-kachan should give a more colourful description here.

Reply:
It could look like the below one. Just added a list of new features provided by the plugin. Feel free to beautify it as needed)
--------------------------------------------------------------
Percona Server for MySQL 8.0.34-26 implements the Audit Log Filter plugin. The plugin provides a set of features which may be used to monitor user activity on the selected server:
- Auditing is done based on filtering rules written in JSON format. Audit events may be filtered based on user or connection, database or table name being accessed, query content, and more.
- Auditing may be dynamically enabled/disabled. Server restart is not required to add or adjust existing filtering rules.
- Filtering rules allow removal of sensitive data from SQL statements written to log file.
- Filtering rules allow blocking of events matching specific criteria (database or table name, user name, and so on).
- Filtering rules may be configured to write to audit log some extra information provided via Query Attributes with each query.
- Audit log files can now be encrypted using AES-256 encryption. The plugin keeps a set of encryption passwords used to encrypt existing log files. External applications may use these passwords to access encrypted logs.
- Audit log files can be compressed to reduce storage space used by log files.
- Automatic rotation and removal of log files may be configured. Outdated log files removal may be done based either on storage space used by log files or log files age.

Reply:
Thank you! Added.